Languages/Python/nice_implementations/tests/qr.txt

```diff
@@ -0,0 +1,162 @@
+i=0
```

Comment:
SHOULD: Remove this

Reply:
Why's that?

it's getting constructed in the print()
If you want I could remove that...

Reply:
Ah..I meant all the file - ideally, we want all the tests to be local to the tests